assets/js/services/helpers.js

```diff
@@ -42,6 +43,15 @@ export function getClassFromIngestSheetStatus(status) {
   return "";
 }
 
+export function isEDTFValid(edtfString) {
+  try {
+    edtf(edtfString).edtf;
+    return true;
+  } catch (e) {
+    return false;
```

Comment:
So the `edtf` method on line 48 throws an Error if its invalid?   Does it give any type of indication of what went wrong?

Reply:
The what went wrong is not indicated as the package throws garbage text if the date is invalid.